docs/installation/manual/installing_karaf_and_alpaca.md

```diff
@@ -0,0 +1,346 @@
+log4j.appender.islandora.maxFileSize=1MB
+log4j.appender.islandora.maxBackupIndex=10
+
+log4j.logger.ca.islandora.camel=INFO, islandora
+```
+
+### Creating a `setenv.sh` Script for Karaf
+
+Similar to Tomcat, our Karaf service is going to rely on a `setenv` shell script to determine environment variables Karaf needs in place when running. For now, this will simply be the path to `JAVA_HOME`, but this also accepts many other parameters you can find in the default `setenv` script.
+
+`/opt/karaf/bin/setenv | karaf:karaf/755`
+```
+#!/bin/sh
+export JAVA_HOME="PATH_TO_JAVA_HOME"
+```
+- `PATH_TO_JAVA_HOME`: This will be the same `JAVA_HOME` we used when installing Tomcat , and can be found using the same method (i.e., still `/usr/lib/jvm/java-8-openjdk-amd64` if that's what it was before).
+
+### Initializing Karaf
+
+We’re going to start Karaf, then run the installer to put our configurations in place and generate a Karaf service. Once these are installed, we’re going to stop Karaf, as from there on out its start/stop management should be handled via that service.
+
+```bash
+sudo -u karaf /opt/karaf/bin/start
+# You may want to wait a bit for Karaf to start.
+# If you're not sure whether or not it's running, you can always run:
+# ps aux | grep karaf
+# to see if the server is up and running.
+/opt/karaf/bin/client feature:install wrapper
+/opt/karaf/bin/client wrapper:install
+/opt/karaf/bin/stop
+```
+
+### Creating and Starting the Karaf Service
+
+Installing the Karaf wrapper generates several service files that can be used on different types of systems. For this example installation on an Ubuntu 18.04 machine, we want to enable the `karaf.service` service so that Karaf is properly started on boot.
+
+```bash
+sudo systemctl enable /opt/karaf/bin/karaf.service
+sudo systemctl start karaf
+```
+
+## Alpaca 1.0.x
+
+### Adding the Required Karaf Repositories
+
+Karaf features can be installed from several different types of sources, but the fastest and easiest way to do so is from existing repository URLs that we can just plug into Karaf to provide us feature lists prepared and ready for installation. Like most interactions with Karaf, we can add these repositories using its built-in `client`.
+
+!!! notice
+    These repositories are updated consistently, and their updates include revised dependency lists. Commonly, when repositories are out of date or otherwise mismatched, feature installation can result in an `Unable to resolve root: missing requirement` error; for this reason, this guide recommends using recently-updated versions of these repositories. That being said, if such errors occur despite installing the latest versions of these features, the maintainer of the features repository should be informed.
+
+For the Karaf features we’re going to install, we need a few different repositories to be added to the list:
+
+```bash
+/opt/karaf/bin/client repo-add mvn:org.apache.activemq/activemq-karaf/ACTIVEMQ_KARAF_VERSION/xml/features
```

Comment:
Any reason we shouldn't use `LATEST` form the versions instead of looking them up? I.e. `mvn:org.apache.activemq/activemq-karaf/LATEST/xml/features` is a working repository ID for me. Similarly with APACHE_CAMEL_VERSION?

Reply:
Thinking about this; we can probably put in `LATEST` as a default for these two and then hop in here and specify if anything ever blows up later

Reply:
FWIW: I've done multiple installs using LATEST and it seems to work fine.